Flash Chat iOS13/Controllers/ThreadListViewController.swift

```diff
@@ -127,32 +127,32 @@ class ThreadListViewController: UITableViewController {
 		let alert = UIAlertController(title: "Are you sure you really want to delete this user?", message: "This can't be undone!", preferredStyle: .alert)
 		let deleteAction = UIAlertAction(title: "Delete", style: .destructive) { action in
 			if let user = Auth.auth().currentUser {
-				user.delete { error in
+				self.database.collection(Constants.FStore.usersCollectionName).getDocuments() { (userQuerySnapshot, error) in
 					if let error = error {
 						AppDelegate.showError(error, inViewController: self)
 					} else {
-						// Account deleted.
-						self.navigationController?.popToRootViewController(animated: true)
-						self.database.collection(Constants.FStore.usersCollectionName).getDocuments() { (userQuerySnapshot, error) in
-							if let error = error {
-								AppDelegate.showError(error, inViewController: self)
-							} else {
-								if let users = userQuerySnapshot?.documents {
-									for userRef in users {
-										if userRef == user {
-											self.database.collection(Constants.FStore.usersCollectionName).document(userRef.documentID).delete { [self]
-												error in
-												if let error = error {
-													AppDelegate.showError(error, inViewController: self)
-												}
-											}
+						if let users = userQuerySnapshot?.documents {
+							for userRef in users {
```

Comment:
🗨️ 
This PR is a specific, discrete bugfix, so we don't have to address this issue in this PR. 

I believe that this flow fetches _all_ users in the users collection. We only want to fetch a single `user` record